lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/util/NetconfUtils.java

```diff
@@ -177,4 +180,13 @@ public static NormalizedNode getUnLockContent(final QName targetDatastore) {
                 .withChild(getTargetNode(targetDatastore)).build();
     }
 
+    public static Optional<Absolute> getAbsolutePathToNNode(final YangInstanceIdentifier yangInstanceIdentifier,
```

Comment:
Maybe try to return Inference directly from here.

Hint: add context to parameters..